### ISSUE TITLE
near clip loss is always zero

### ISSUE THREAD

**Issue by sp9103** [sunset1995/DirectVoxGO#26]
When calculating nearclip_loss in scene_rep_reconstruction, it always seems to 0.

```
if cfg_train.weight_nearclip > 0:
    near_thres = data_dict['near_clip'] / model.scene_radius[0].item()
    near_mask = (render_result['t'] < near_thres)
    density = render_result['raw_density'][near_mask]
    if len(density):
        nearclip_loss = (density - density.detach()).sum()
        loss += cfg_train.weight_nearclip * nearclip_loss
```

What was the original intent of that code?

**Comment by sunset1995** [comment]
I don't want this loss to affect total loss for easier inspecting the trend between different runs.
In the backward pass, it is actually equivalent to `density.sum()`.

**Comment by sp9103** [comment]
Thanks :)
